### ISSUE TITLE
Whitelist of plain URLs to iframe

### ISSUE THREAD

**Issue by plexish** [StampyAI/stampy-ui#140]
Would be super nice to have https://aisafety.world/map iframed on the main page at least, maybe https://coda.io/@alignmentdev/alignmentecosystemdevelopment in another answer, having that list expandable might prove valuable if not too complex.

If a full plain URL from the whitelisted domains is in the markdown, it should be transformed into a large (~screen height, full answer width) iframe of that URL.

- [ ] Airtable
- [ ] Coda
- [ ] https://aisafety.world/

**Comment by jrhender** [comment]
I'm going to take a look at this as a part of the EA London Hackathon 🚀

**Comment by jrhender** [comment]
Here's an example:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/4407464/218254771-09341fa7-ee5b-4ce1-af7d-9e327d2ac567.png">

In the above, https://google.com should render as an iframe of https://google.com

**Comment by jrhender** [comment]
Another reason for having the whitelist (as mentioned in the issue description) is that it might be good to test if sites have set X-Frame-Options such that their site can be loaded in an iframe: https://stackoverflow.com/questions/27358966/how-to-set-x-frame-options-on-iframe

**Comment by jrhender** [comment]
Discussing with @plexish , the "parser" tag (currently applied to this issue) may be inaccurate as it implies that the url -> iframe transformation should be done prior to the data being served to the frontend from coda. This is not ideal because the coda data is also consumed by Discord which cannot render iframes.
